Sources/Filters/General/AppendPolyData/index.js

```diff
@@ -4,9 +4,9 @@ import vtkPoints        from 'vtk.js/Sources/Common/Core/Points';
 import vtkPolyData      from 'vtk.js/Sources/Common/DataModel/PolyData';
 import { VtkDataTypes } from 'vtk.js/Sources/Common/Core/DataArray/Constants';
 
-import Constants        from 'vtk.js/Sources/Filters/General/AppendPolyData/Constants';
+import Constants        from 'vtk.js/Sources/Filters/General/Constants';
```

Comment:
Is that constant supposed to be at the module level `Filters/General`?
I think they should belong to a class like `PointSets`?

Reply:
Well, technically it is supposed to be an algorithm enum like [vtkAlgorithm::DesiredOutputPrecision](https://www.vtk.org/doc/nightly/html/classvtkAlgorithm.html#ac2bf3c779aa60908da5bc3df9b066db6) 

That's why I put it at the module level so that all ***general filters*** have access to it.

Reply:
I see, but we don't have a base class for vtkAlgorithm... ;-)

Then maybe we want to add a class vtkAlgorithm which only expose its constants. Is there any other constants in that class/enum? If that's the only one, I'm wondering if we should find a better place...